src/modules/roc_packet/target_stdio/roc_packet/print.cpp

```diff
@@ -35,7 +35,8 @@ void print(const Packet& p, int flags) {
     }
 
     if (p.fec()) {
-        fprintf(stderr, "fec: blk=%lu payload=%lu\n", (unsigned long)p.fec()->blknum,
+        fprintf(stderr, "fec: sbn=%lu payload=%lu\n",
+                (unsigned long)p.fec()->source_block_number,
```

Comment:
Could you please also add source block length?

Reply:
Done

Reply:
"sbn_len" -> "sb_len", or maybe "sbl" or "sblen". I suggest "sblen".

Reply:
Done